test/config/config.go

```diff
@@ -32,4 +33,6 @@ func (c *CiliumTestConfigType) ParseFlags() {
 		"Provision Vagrant boxes and Cilium before running test")
 	flag.BoolVar(&c.HoldEnvironment, "cilium.holdEnvironment", false,
 		"On failure, hold the environment in its current state")
+	flag.BoolVar(&c.ShowCommands, "cilium.showCommands", false,
```

Comment:
Update `Documentation/contributing.rst` under 'Available CLI Options' to reflect this new change.

Change description to say "output which commands are ran to stdout"